styles/floating-images.css

```diff
@@ -0,0 +1,99 @@
+/* block specific CSS goes here */
+
+main .section.image-right .default-content-wrapper {
+    display: flex;
+    flex-direction: row-reverse;
+    max-width: 100vw;
+    height: 530px;
+    padding: 0  0 0 15px;
```

Comment:
So, i added a padding to all `.section` items under main. The left/right padding on these probably won't be needed.

Reply:
updated